react-app/src/store/session.js

```diff
@@ -45,14 +45,15 @@ export const login = (email, password) => async (dispatch) => {
 		const data = await response.json();
 		dispatch(setUser(data));
 		return null;
-	} else if (response.status < 500) {
-		const data = await response.json();
-		if (data.errors) {
-			return data.errors;
-		}
-	} else {
-		return ["An error occurred. Please try again."];
-	}
+	} 
```

Comment:
try to avoid changing reducer files or backend api routes without mentioning to the rest of the team since we already tested this file and pushed it to dev
how come this code is commented out?